config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
+  mount MagicLamp::Genie, at: '/magic_lamp' if defined?(MagicLamp)
```

Comment:
I hope there's a better way.  This is copied from Magic Lamp's README.  I'm getting it to work by the book first before making it nice.  I'll add this to TODO so I don't forget and leave it here.